DomainContent/AvatarStore/attachmentItemScript.js

```diff
@@ -7,37 +7,60 @@
 //  Distributed under the Apache License, Version 2.0.
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
+/* global Render, Selection */
+
+
+
 (function() {
-    var shared = Script.require('./attachmentZoneShared.js');
-    
+    var highlightToggle = false;
+  
     var GRAB_SOUND = SoundCache.getSound(Script.resolvePath('sounds/sound1.wav'));
     var ATTACH_SOUND = SoundCache.getSound(Script.resolvePath('sounds/sound2.wav'));
     var DETACH_SOUND = SoundCache.getSound(Script.resolvePath('sounds/sound7.wav'));
-
+    var HIGHLIGHT = Script.require('./ExternalOutlineConfig.js');
+    var SHARED = Script.require('./attachmentZoneShared.js');
```

Comment:
Please replace 'shared' with 'SHARED' in the rest of this file since it has been changed here.